tests/data/server-content/www.eclipse.org/downloads/index.html

```diff
@@ -0,0 +1,896 @@
+              <img src="//bs.serving-sys.com/BurstingPipe/adServer.bs?cn=bsr&FlightID=15916521&Page=&PluID=0&Pos=616231926" width=32 height=32 alt="Click Here" border=0>
+            </a>
+          </noscript>
+        </div>
+        <div class="col-sm-13 text xs-sm-text-center">
+          <h3>
+            <a href="//bs.serving-sys.com/BurstingPipe/adServer.bs?cn=tf&c=20&mc=click&pli=15933613&PluID=0&ord=[timestamp]" target="_blank">Run your Apps on the Cloud              <img src="" width=1 height=1 alt=" " border=0>
+            </a>
+          </h3>
+          <p>Want Eclipse in the cloud? IBM Bluemix makes it easy. Sign up to begin building today!</p>
+        </div>
+        <div class="col-sm-8 download">
+          <div class="col-sm-9 downloadLink-icon"><i class="fa fa-download"></i></div>
+          <div class="col-sm-15 downloadLink-content">
+            <div class="text-center">
+              <p>
+                <a class="orange" href="//bs.serving-sys.com/BurstingPipe/adServer.bs?cn=tf&c=20&mc=click&pli=15933613&PluID=0&ord=[timestamp]" target="_blank">
+                  <i class="fa fa-star"></i><br/>Promoted<br/>Download</a><br/></p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+        <div class="package-row clearfix" data-Filter-packages="Git Team Provider, Java Development Tools, Maven Integration for Eclipse, Mylyn Task List, Code Recommenders Tools for Java Developers, WindowBuilder Core, XML Editors and Tools">
+      <div class="row vertical-align-min-md">
+        <div class="col-sm-3 icon">
+          <img src="/downloads/images/java.png" width="42" height="42" alt="Eclipse IDE for Java Developers"/>
+        </div>
+        <div class="col-sm-13 text xs-sm-text-center">
+          <h3>
+            <a href="http://www.eclipse.org/downloads/packages/eclipse-ide-java-developers/mars1" title="The essential tools for any Java developer, including a Java IDE, a Git client, XML Editor, Mylyn, Maven integration and WindowBuilder">
+              Eclipse IDE for Java Developers            </a>
+          </h3>
+
+
+          <ul class="list-inline">
+            <li class="download-counts">165 MB</li>
+            <li class="download-counts">908,477 DOWNLOADS</li>
+          </ul>
+          <p>The essential tools for any Java developer, including a Java IDE, a Git client, XML Editor, Mylyn, Maven integration and WindowBuilder...</p>
+        </div>
+        <div class="col-sm-8 download">
+          <div class="col-sm-9 downloadLink-icon"><i class="fa fa-download"></i></div>
+          <div class="col-xs-24 col-sm-15 downloadLink-content">
+            <div class="text-center">
+              <p class="margin-bottom">Linux</p>
+              <ul class="list-inline">
+                                  <li><a class="downloadLink" href="download.php?file=/technology/epp/downloads/release/version/point_release/eclipse-java-linux-gtk.tar.gz" title="Download">32 bit</a></li>
```

Comment:
Yeah, that should work, there is a redirect feature in the mock server helping with this, I just need to give a second look to it and run the medium test to ensure it works well :)